.pre-commit-config.yaml

```diff
@@ -68,6 +68,8 @@ repos:
           - packaging
           - tomli
           - types-pkg_resources
+            # for python>=3.11
```

Comment:
`python<3.11`?

Reply:
on python<3.11 it's required by `install_depends` and will be installed, but on python>=3.11 it won't be, and so mypy will fail typechecking exceptiongroup if the linting is run in a default environment of 3.11